examples/pytorch/speech-recognition/requirements.txt

```diff
@@ -1,4 +1,4 @@
-datasets >= 1.13.3
+datasets >= 1.18.0
```

Comment:
The upgrade is needed to support accessing `dl_manager.download_config` https://huggingface.co/datasets/mozilla-foundation/common_voice_8_0/blob/main/common_voice_8_0.py#L144

Saw some reports about that on Discord last week :)